configuration/pih/htmlforms/hiv/section-hiv-history.xml

```diff
@@ -711,6 +721,42 @@
         </section>
     </includeIf>
 
+    <section id="contacts" sectionTag="fieldset" headerTag="legend" headerStyle="title"
+            headerCode="pihcore.contacts">
+        <repeat with="['1'],['2'],['3'],['4'],['5']">
+            <div class="section-container mb-4 flex-questions">
+                <obsgroup groupingConceptId="PIH:Contact construct">
+                    <div>
+                        <obs conceptId="PIH:RELATIONSHIP OF RELATIVE TO PATIENT"
+                         labelCode="pihcore.contactRelationship"
+                         answerConceptIds="PIH:PARTNER OR SPOUSE,PIH:GUARDIAN RELATIONSHIP TO CHILD,PIH:SIBLING,PIH:OTHER RELATIVE"/>
+                    </div>
+                    <div>
+                        <obs conceptId="PIH:FIRST NAME" labelCode="pihcore.firstName"/>
+                    </div>
+                    <div>
+                        <obs conceptId="PIH:LAST NAME" labelCode="pihcore.lastName"/>
+                    </div>
+                    <div>
+                        <obs conceptId="PIH:AGE" labelCode="pihcore.age"/>
+                    </div>
+                    <div>
+                        <obs conceptId="CIEL:163533"
+                             labelCode="pihcore.dead"
+                             answerConceptIds="PIH:ALIVE,PIH:DEATH,PIH:UNKNOWN"
```

Comment:
The paper form is different, but I mostly like this.  I would suggest using answerCodes=yes/DEATH, no/ALIVE, unknown/unknown.  This keeps it similar to the paper, but adding unknown.  Otherwise it looks odd with the header or "death" with dead.   Too knit-picky?

Reply:
Not at all, that's great feedback, I agree. Fixed it up. I'd also messed up the formatting—I'd intended for these to be radio buttons rather than dropdowns. I've fixed that as well.